test/e2e/shared/identity.go

```diff
@@ -60,38 +59,33 @@ func SetupStaticCredentials(ctx context.Context, namespace *corev1.Namespace, e2
 
 	id := &infrav1.AWSClusterStaticIdentity{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      idName,
-			Namespace: namespace.Name,
+			Name: idName,
 		},
 		Spec: infrav1.AWSClusterStaticIdentitySpec{
 			SecretRef: credsSecretName,
 			AWSClusterIdentitySpec: infrav1.AWSClusterIdentitySpec{
-				AllowedNamespaces: &infrav1.AllowedNamespaces{
-					NamespaceList: []string{namespace.Name},
-				},
+				AllowedNamespaces: &infrav1.AllowedNamespaces{},
```

Comment:
I think it would be better to keep it limited to the namespace of the test

Reply:
I debated about it. If we want to use the namespace of the test which is created at the start of the each test run, it will be better to move back SetupStaticCredentials to each test as we don't know the test namespace in advance. 
Any thoughts?

Reply:
Ah yes, good point @pydctw . Ignore me :smile: